src/Views/BucketPanel/EntryDetail.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useState, useEffect } from "react";
+import { useHistory, useParams } from "react-router-dom";
+import { Client, EntryInfo, TokenPermissions } from "reduct-js";
+import {
+  Table,
+  Typography,
+  DatePicker,
+  Button,
+  InputNumber,
+  Checkbox,
+} from "antd";
+import { ReadableRecord } from "reduct-js/lib/cjs/Record";
+import { DownloadOutlined } from "@ant-design/icons";
+import EntryCard from "../../Components/Entry/EntryCard";
+import "./EntryDetail.css";
+
+// @ts-ignore
+import prettierBytes from "prettier-bytes";
+
+interface Props {
+  client: Client;
+  permissions?: TokenPermissions;
+}
+
+export default function EntryDetail(props: Readonly<Props>) {
+  const { bucketName, entryName } = useParams() as {
+    bucketName: string;
+    entryName: string;
+  };
+  const history = useHistory();
+  const [isLoading, setIsLoading] = useState(false);
+  const [records, setRecords] = useState<ReadableRecord[]>([]);
+  const [start, setStart] = useState<bigint | undefined>(undefined);
+  const [end, setEnd] = useState<bigint | undefined>(undefined);
+  const [limit, setLimit] = useState<number | undefined>(10);
+  const [showUnix, setShowUnix] = useState(false);
+  const [entryInfo, setEntryInfo] = useState<EntryInfo>();
+
+  const getRecords = async (start?: bigint, end?: bigint, limit?: number) => {
+    setIsLoading(true);
+    setRecords([]);
+    try {
+      const bucket = await props.client.getBucket(bucketName);
+      for await (const record of bucket.query(entryName, start, end, {
+        limit: limit,
```

Comment:
use the head options, to avoid downloading the content of records: https://github.com/reductstore/reduct-js/blob/main/src/messages/QueryEntry.ts#L67
```suggestion
        limit: limit,
        head: true, // we fetch only meta information without the content
```
```suggestion
        limit: limit,
```